src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala

```diff
@@ -86,7 +86,7 @@ case class ApolloSourceGenerator(
       // render the document into the query object.
       // replacing single $ with $$ for escaping
       val escapedDocumentString =
-        operation.original.renderPretty.replaceAll("\\$", "\\$\\$")
+        operation.original.renderPretty
```

Comment:
Not sure why you removed that (I don't mind) but it seems like the comment is now wrong.

Reply:
Hi Felix. Great to meet you here, too :-) 
The replaceAll of the $ is still required in the next lines (to escape the fragment code).
I will have a look why there is still required (and therefore I left this comment, but
you are right, that it's not in the right place. It should be moved into next comment.)

Reply:
I had a look and removed the replaceAll from the fragment part - it's not required now, otherwise it breaks the test case that I've added to validate.